sbncode/Calibration/TrackCaloSkimmer_module.cc

```diff
@@ -18,6 +18,9 @@
 #include "sbncode/CAFMaker/FillTrue.h"
 #include "sbncode/CAFMaker/RecoUtils/RecoUtils.h"
 
+// C/C++ includes
+#include <filesystem>
```

Comment:
We found out in the other PR that `<filesystem>` doesn't work properly under clang. I think you will have to use my old `basename()` implementation instead.